src/docs/regulatory_predicted_molecular_consequence.md

```diff
@@ -0,0 +1,44 @@
+# Regulatory Predicted Molecular Consequence
+
+| Field             | Type            | Description
+|-------------------|-----------------|---------------------
+|allele_name|string|Data-derived allele name
+|stable_id|string|Feature stable id
+|feature_type|OntologyTermMetadata|SO term for feature type (eg. transcript regulatory feature)
+|sub_feature_type|OntologyTermMetadata|SO term for feature type (eg. enhancer, CTCF)
+|consequences|array of OntologyTermMetadata|SO terms for predicted impact ( eg regulatory_region_variant)
+|regulatory_feature_relative_location|VariantRelativeLocation or null|Relative location on regulatory feature
+
+
+```json
+"name": "rs1761454157",
+"alleles":  [
+    {
+    "name": "5:173235220:G:A",
+    "predicted_molecular_consequences": {},
+    "regulatory_predicted_molecular_consequences": {
+        "allele_name": "A",
+        "stable_id": "ENSR5_94JQHZ",
+        "feature_type": {
+            "value": "Regulatory Feature"
+            },
+        "sub_feature_type": {
+            "value": "Promoter"
```

Comment:
Our features names in the new world (GFF3 file) are standardised by SO, which means they are lower case.  For this field, can we use SO (or a subset) as the controlled vocabulary? Additionally, we plan to introduce sub-classes for features in the future e.g. different types of _promoters_. Therefore, it might be useful to name these JSON fields different.

Reply:
@ilsley - would a promotor type etc be similar in concept to a transcript biotype?

Reply:
BTW, would this not be a `feature_subtype` rather than a `sub_feature_type`? I.e. a subtype of a feature rather than a type of a subfeature?

Reply:
Following [DMC meeting](https://www.ebi.ac.uk/seqdb/confluence/display/ENS/DMC+-+2024-12-05), it is decided that subclasses are handled at feature level. 

`promoter` (as per SO naming convention) is now captured in feature_type as feature_type intends to capture the type coming from GFF3 files